src/Reflection/ReflectionClass.php

```diff
@@ -331,6 +331,18 @@ public function hasProperty($name)
     }
 
     /**
+     * Protected API to add properties at runtime (used in ReflectionObject)
+     *
+     * @param ReflectionProperty $property
+     */
+    protected function addProperty(ReflectionProperty $property)
```

Comment:
Yes:

```php
$foo = new \stdClass();
$foo->bar = 'baz';
```
If you create a `ReflectionObject` using `$foo` then `bar` is a runtime property.

`ReflectionObject` can't access `ReflectionClass` properties as they're private. However, can modify them to simply append them as we overwrite all the methods anyway.